css/main.css

```diff
@@ -14,50 +14,53 @@
     flex-direction: column;
```

Comment:
[OPTIONAL] 
- [ ] You can consider adding some space between your elements for a better user experience on your site.
As you can see how elements are not spaced correctly on your site :point_down: 
![image](https://user-images.githubusercontent.com/97100091/196744443-5a19e996-ba47-49f8-8e68-7cb3108c2cc6.png)
- [ ] The logo is not really spaced with the content